src/ansys/fluent/core/solver/flobject.py

```diff
@@ -243,6 +244,33 @@ def set_state(self, state: StateT = None, **kwargs):
         else:
             return self.flproxy.set_var(self.path, self.to_scheme_keys(state))
 
+    def find_child(self, identifier="*"):
+        """Lists path of all the child objects matching an identifier."""
+        _list_of_children = []
+        SettingsBase._list_children(self.__class__, identifier, [], _list_of_children)
+        return _list_of_children
+
+    @staticmethod
+    def _list_children(cls, identifier, path, _list_of_children):
+        if issubclass(cls, NamedObject):
+            if hasattr(cls.child_object_type, "child_names"):
+                SettingsBase._get_child_path(
+                    cls.child_object_type, path, identifier, _list_of_children
+                )
+        if issubclass(cls, Group):
+            SettingsBase._get_child_path(cls, path, identifier, _list_of_children)
```

Comment:
Shouldn't the above be defined in the derived classes?

Why is `ListObject` missing here?

Reply:
ListObject has been included as well. Thank you.